modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/dto/v1_0/converter/test/ObjectEntryDTOConverterTest.java

```diff
@@ -110,6 +124,23 @@ public void testToDTO() throws Exception {
 
 		Map<String, Object> properties = objectEntry.getProperties();
 
+		List<ListEntry> listEntries = (List<ListEntry>)properties.get(
+			"multiselectPicklist");
+
+		Assert.assertNotNull(listEntries);
+		Assert.assertEquals(listEntries.toString(), 2, listEntries.size());
+
+		Assert.assertEquals(
+			"listTypeEntryKey1",
+			listEntries.get(
+				0
+			).getKey());
+		Assert.assertEquals(
+			"listTypeEntryKey2",
+			listEntries.get(
+				1
+			).getKey());
```

Comment:
why did you add this empty between the Asserts?
I dont think we should have an empty line.

Reply:
It was unintentional. Removed.